cvat/apps/dataset_manager/tests/test_formats.py

```diff
@@ -484,6 +576,498 @@ def test_frames_outside_are_not_generated(self):
+                self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+                # upload annotations
+                response = self._upload_file(url_upload, binary_file, user)
+
+                if user is None:
+                    self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+                    # check for missing annotations
+                    response = self._get_request(url, self.admin)
+                    self.assertEqual(response.status_code, status.HTTP_200_OK)
+                    self.assertEqual(len(response.data["tags"]), 0)
+                    self.assertEqual(len(response.data["shapes"]), 0)
+                else:
+                    self.assertEqual(response.status_code, status.HTTP_202_ACCEPTED)
+                    # check for presence annotations
+                    response = self._get_request(url, self.admin)
+                    self.assertEqual(response.status_code, status.HTTP_200_OK)
+                    self.assertNotEqual(len(response.data["tags"]), 0)
+                    self.assertNotEqual(len(response.data["shapes"]), 0)
+
+    def test_api_v1_tasks_annotations_dump_and_dataset_export_with_datumaro(self):
+        test_name = self._testMethodName
+        importer_format_name = {
+            'COCO 1.0': 'coco',
+            'CVAT for images 1.1': 'cvat',
+            'CVAT for video 1.1': 'cvat',
+            'Datumaro 1.0': 'datumaro_project',
+            'LabelMe 3.0': 'label_me',
+            'MOT 1.1': 'mot_seq',
+            'MOTS PNG 1.0': 'mots_png',
+            'PASCAL VOC 1.1': 'voc',
+            'Segmentation mask 1.1': 'voc',
+            'TFRecord 1.0': 'tf_detection_api',
+            'YOLO 1.1': 'yolo',
+            'ImageNet 1.0': 'imagenet_txt',
+            'CamVid 1.0': 'camvid',
+            'WiderFace 1.0': 'wider_face',
+            'VGGFace2 1.0': 'vgg_face2',
+        }
+
+        # get formats
+        dump_formats = dm.views.get_export_formats()
+
+        for dump_format in dump_formats:
+            if dump_format.ENABLED:
+                dump_format_name = dump_format.DISPLAY_NAME
+
+                if dump_format_name != "Datumaro 1.0":
+                    continue
+                # TODO skip failed formats
+                if dump_format_name in [
+                    "CVAT for video 1.1",
+                    "YOLO 1.1",
+                    "ImageNet 1.0",
+                    "Datumaro 1.0",
+                ]:
+                    continue
+
+                print("dump_format_name:", dump_format_name)
+
+                # create task
+                images = self._generate_task_images(3)
+                task = self._create_task(tasks["main"], images)
+
+                 # create annotations
+                if dump_format_name in ["MOT 1.1", "MOTS PNG 1.0", "PASCAL VOC 1.1", "Segmentation mask 1.1", "TFRecord 1.0", "YOLO 1.1", "ImageNet 1.0", "WiderFace 1.0", "VGGFace2 1.0"]:
```

Comment:
Strive to keep line length under 80 characters.

Reply:
Done.